qualtran/bloqs/block_encoding.py

```diff
@@ -336,39 +536,140 @@ def _black_box_select() -> BlackBoxSelect:
 
 
 @bloq_example
-def _black_box_block_bloq() -> BlackBoxBlockEncoding:
-    from qualtran.bloqs.block_encoding import BlackBoxBlockEncoding, BlackBoxPrepare, BlackBoxSelect
+def _lcu_block_bloq() -> LCUBlockEncoding:
+    from qualtran.bloqs.hubbard_model import PrepareHubbard, SelectHubbard
+
+    # 3x3 hubbard model U/t = 4
+    dim = 3
+    select = SelectHubbard(x_dim=dim, y_dim=dim)
+    U = 4
+    t = 1
+    prepare = PrepareHubbard(x_dim=dim, y_dim=dim, t=t, u=U)
+    N = dim * dim * 2
+    qlambda = 2 * N * t + (N * U) // 2
+    lcu_block_bloq = LCUBlockEncoding(select=select, prepare=prepare, alpha=qlambda, epsilon=0.0)
+    return lcu_block_bloq
```

Comment:
lol can we ommit `_bloq` suffix here and below? black box block bloq is admittedly very funny but maybe not in a helpful way :)

Reply:
Can you say it 5 times quickly? I knew I pushed it too far this time, I will retire black_box_block_bloq, a career highlight.

Reply:
RIP